### DISCUSSION TITLE
Unit decibel

### DISCUSSION THREAD

**Discussion by maverick56** [josdejong/mathjs#2911] [Ideas]
Does decibel impremented as unit ?

I saw here it's done (2019.06.08) ; 
https://mathjs.org/history.html

But i don't see it on code or on documentation here : 
https://mathjs.org/docs/datatypes/units.html
https://github.com/josdejong/mathjs/blob/develop/src/type/unit/Unit.js

If doesn't existe, is it planned ?

tks a lot

**Comment by josdejong** [comment]
It was removed again due to complications, see this discussion: https://github.com/josdejong/mathjs/pull/1518#issuecomment-494226286
